SimplifiedTests/NYPLAxisTests/NYPLAxisMetadataServiceTests.swift

```diff
@@ -14,109 +14,88 @@ class NYPLAxisMetadataServiceTests: XCTestCase {
       .appendingPathComponent(keysProvider.containerDownloadEndpoint)
   }()
-  
+
   lazy private var metadataService: NYPLAxisMetadataService = {
     return NYPLAxisMetadataService(
       axisItemDownloader: itemDownloader, axisKeysProvider: keysProvider,
       baseURL: baseURL, parentDirectory: downloadsDirectory)
   }()
-  
-  func testMetadataServiceShouldNotContinueUponDownloadFailure() {
-    let leaveExpecation = self.expectation(
-      description: "Metadata service should leave dispatch group upon failure")
-    let encryptonDownloadExpectation = self.expectation(
-      description: "Metadata service should attempt to download encryption.xml")
-    
-    contentDownloader.mockDownloadFailure()
-    contentDownloader.didReceiveRequestForUrl = {
-      if $0 == self.containerURL {
-        XCTFail()
-      }
-      if $0 == self.encryptionURL {
-        encryptonDownloadExpectation.fulfill()
-      }
-    }
-    
-    metadataService.downloadContent()
-    itemDownloader.dispatchGroup.notify(queue: .global()) {
-      leaveExpecation.fulfill()
-    }
-    
-    waitForExpectations(timeout: 4, handler: nil)
-  }
-  
-  func testMetadataServiceShouldNotContinueUponWriteFailure() {
-    let leaveExpecation = self.expectation(
-      description: "Metadata service should leave dispatch group upon failure")
+
+
+  func testMetadataServiceShouldDownloadContainerUponEncryptionDownloadSuccess() {
+    let containerRequestExpecatation = self.expectation(
+      description: "Metadata service should download container upon encryption download success")
     let encryptonDownloadExpectation = self.expectation(
       description: "Metadata service should attempt to download encryption.xml")
-    
+
     contentDownloader.mockDownloadSuccess()
-    assetWriter.mockingFailure()
+    assetWriter.mockingSuccess()
     contentDownloader.didReceiveRequestForUrl = {
       if $0 == self.containerURL {
-        XCTFail()
+        containerRequestExpecatation.fulfill()
       }
       if $0 == self.encryptionURL {
         encryptonDownloadExpectation.fulfill()
       }
     }
     
-    metadataService.downloadContent()
-    itemDownloader.dispatchGroup.notify(queue: .global()) {
-      leaveExpecation.fulfill()
-    }
-    
+    let aggregator = NYPLAxisTaskAggregator()
+    aggregator
+      .addTasks(metadataService.downloadMetadataTasks())
+      .run()
+      .onCompletion { _ in }
+
     waitForExpectations(timeout: 4, handler: nil)
   }
   
-  func testMetadataServiceShouldDownloadContainerUponEncryptionDownloadSuccess() {
-    let requestExpecatation = self.expectation(
-      description: "Metadata service should download container upon encryption download success")
-    let leaveExpecation = self.expectation(
-      description: "Metadata service should leave dispatch group upon success")
+  func testMetadataServiceShouldNotDownloadContainerUponEncryptionDownloadFailure() {
+    let containerRequestExpecatation = self.expectation(
```

Comment:
Another typo

Reply:
Fixed.
Note to self: install spell check on xcode